controllers/subscriptions.go

```diff
@@ -351,34 +351,54 @@ func EnsureVendorCsv(cli client.Client, csvName string) (*operatorv1alpha1.Clust
 		}
 		return nil, err
 	}
+
+	var replicas int32
+	var isProvider bool
+
+	if strings.HasPrefix(csvName, "ocs-client-operator") {
+		isProvider, err = isProviderMode(cli)
+		if err != nil {
+			return nil, err
+		}
+
+		if isProvider {
+			replicas = 1
+		}
+
+		// Patch to set deployments replicas
+		patchStr := fmt.Sprintf(`{"spec":{"replicas":%d}}`, replicas)
+		patch := client.RawPatch(types.MergePatchType, []byte(patchStr))
+
+		// Shut down the OCS client operator pods in non provider mode
+		for _, deployment := range []string{"ocs-client-operator-console", "ocs-client-operator-controller-manager"} {
```

Comment:
We need to perform this operation after we set replicas in csv.

Reply:
Done